### PR TITLE
Add support for Iron165-H

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

Adds support for the Iron165-H (hotswap) PCB. Since right now there is only a `iron165r2` folder, the Iron165-S (solderable) was moved to a new folder `iron165r2_s` so that `iron165r2_h` would be used for the hotswap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by sigprof** [comment]
Renaming a keyboard would need to go through the `develop` branch.

**Comment by drashna** [comment]
And should be included in the `data/mappings/keyboard_aliases.json`

**Comment by Gondolindrim** [comment]
> Renaming a keyboard would need to go through the `develop` branch.

[9234585](https://github.com/qmk/qmk_firmware/pull/20617/commits/9234585a593cbbf4d3a587350bb8037820b93439) pushes changes against develop. The change on RGB configuration from `rgblight.pin` to `ws2812.pin` was made.

**Comment by Gondolindrim** [comment]
> 05a37ff

Done in [05a37ff](https://github.com/qmk/qmk_firmware/pull/20617/commits/05a37ffd5b1be37746a5f7c4928fb8436af0c2ea) .

**Review by drashna — APPROVED** [review]

File: data/mappings/keyboard_aliases.hjson [hunk 1]
```diff
@@ -257,6 +257,15 @@
...
     },
+    "iron165r2_s_f411":{
+        "target": "smithrune/iron165r2_s/f411"
+    },
+    "iron165r2_s_f072":{
+        "target": "smithrune/iron165r2_s/f072"
+    },
+    "iron165r2_h_f072":{
+        "target": "smithrune/iron165r2_h"
+    },
```
Comment (noroadsleft) [inline data/mappings/keyboard_aliases.hjson:268]:
```suggestion
    "smithrune/iron165r2/f411":{
        "target": "smithrune/iron165r2_s/f411"
    },
    "smithrune/iron165r2/f072":{
        "target": "smithrune/iron165r2_s/f072"
    },
```

The format for this is:

```
    "<old keyboard name>": {
        "target": "<new keyboard name>"
    },
```

File: keyboards/smithrune/iron165r2_h/info.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                {"label":"End",       "matrix":[ 3,14], "x":15,    "y":3          }, 
+                {"label":"Ctrl",      "matrix":[ 4, 0], "x":0,     "y":4, "w":1.5 }, 
+                {"label":"Win",       "matrix":[ 4, 1], "x":1.5,   "y":4          }, 
+                {"label":"Alt",       "matrix":[ 4, 2], "x":2.5,   "y":4, "w":1.5 }, 
+                {"label":"Space",     "matrix":[ 4, 6], "x":4,     "y":4, "w":7   }, 
+                {"label":"Alt",       "matrix":[ 4,10], "x":11,    "y":4, "w":1.5 }, 
+                {"label":"Left",      "matrix":[ 4,11], "x":13,    "y":4          }, 
+                {"label":"Down",      "matrix":[ 4,12], "x":14,    "y":4          }, 
+                {"label":"Right",     "matrix":[ 4,13], "x":15,    "y":4          },
+                {"label":"Del",       "matrix":[ 4,14], "x":15,    "y":0          }
```
Comment (noroadsleft) [inline keyboards/smithrune/iron165r2_h/info.json:106]: Moves Delete to the top row.

```suggestion
                {"label":"Del",       "matrix":[ 4,14], "x":15,    "y":0          } 
                {"label":"Tab",       "matrix":[ 1, 0], "x":0,     "y":1, "w":1.5 }, 
                {"label":"Q",         "matrix":[ 1, 1], "x":1.5,   "y":1          }, 
                {"label":"W",         "matrix":[ 1, 2], "x":2.5,   "y":1          }, 
                {"label":"E",         "matrix":[ 1, 3], "x":3.5,   "y":1          }, 
                {"label":"R",         "matrix":[ 1, 4], "x":4.5,   "y":1          }, 
                {"label":"T",         "matrix":[ 1, 5], "x":5.5,   "y":1          }, 
                {"label":"Y",         "matrix":[ 1, 6], "x":6.5,   "y":1          }, 
                {"label":"U",         "matrix":[ 1, 7], "x":7.5,   "y":1          }, 
                {"label":"I",         "matrix":[ 1, 8], "x":8.5,   "y":1          }, 
                {"label":"O",         "matrix":[ 1, 9], "x":9.5,   "y":1          }, 
                {"label":"P",         "matrix":[ 1,10], "x":10.5,  "y":1          }, 
                {"label":"{",         "matrix":[ 1,11], "x":11.5,  "y":1          }, 
                {"label":"}",         "matrix":[ 1,12], "x":12.5,  "y":1          }, 
                {"label":"|",         "matrix":[ 1,13], "x":13.5,  "y":1, "w":1.5 }, 
                {"label":"Page Up",   "matrix":[ 1,14], "x":15,    "y":1          }, 
                {"label":"Caps Lock", "matrix":[ 2, 0], "x":0,     "y":2, "w":1.75}, 
                {"label":"A",         "matrix":[ 2, 1], "x":1.75,  "y":2          }, 
                {"label":"S",         "matrix":[ 2, 2], "x":2.75,  "y":2          }, 
                {"label":"D",         "matrix":[ 2, 3], "x":3.75,  "y":2          }, 
                {"label":"F",         "matrix":[ 2, 4], "x":4.75,  "y":2          }, 
                {"label":"G",         "matrix":[ 2, 5], "x":5.75,  "y":2          }, 
                {"label":"H",         "matrix":[ 2, 6], "x":6.75,  "y":2          }, 
                {"label":"J",         "matrix":[ 2, 7], "x":7.75,  "y":2          }, 
                {"label":"K",         "matrix":[ 2, 8], "x":8.75,  "y":2          }, 
                {"label":"L",         "matrix":[ 2, 9], "x":9.75,  "y":2          }, 
                {"label":":",         "matrix":[ 2,10], "x":10.75, "y":2          }, 
                {"label":"\"",        "matrix":[ 2,11], "x":11.75, "y":2          }, 
                {"label":"Enter",     "matrix":[ 2,13], "x":12.75, "y":2, "w":2.25}, 
                {"label":"Page Down", "matrix":[ 2,14], "x":15,    "y":2          }, 
                {"label":"Shift",     "matrix":[ 3, 0], "x":0,     "y":3, "w":2.25},
                {"label":"Z",         "matrix":[ 3, 2], "x":2.25,  "y":3          }, 
                {"label":"X",         "matrix":[ 3, 3], "x":3.25,  "y":3          }, 
                {"label":"C",         "matrix":[ 3, 4], "x":4.25,  "y":3          }, 
                {"label":"V",         "matrix":[ 3, 5], "x":5.25,  "y":3          }, 
                {"label":"B",         "matrix":[ 3, 6], "x":6.25,  "y":3          }, 
                {"label":"N",         "matrix":[ 3, 7], "x":7.25,  "y":3          }, 
                {"label":"M",         "matrix":[ 3, 8], "x":8.25,  "y":3          }, 
                {"label":"<",         "matrix":[ 3, 9], "x":9.25,  "y":3          }, 
                {"label":">",         "matrix":[ 3,10], "x":10.25, "y":3          }, 
                {"label":"?",         "matrix":[ 3,11], "x":11.25, "y":3          }, 
                {"label":"Shift",     "matrix":[ 3,12], "x":12.25, "y":3, "w":1.75}, 
                {"label":"Up",        "matrix":[ 3,13], "x":14,    "y":3          }, 
                {"label":"End",       "matrix":[ 3,14], "x":15,    "y":3          }, 
                {"label":"Ctrl",      "matrix":[ 4, 0], "x":0,     "y":4, "w":1.5 }, 
                {"label":"Win",       "matrix":[ 4, 1], "x":1.5,   "y":4          }, 
                {"label":"Alt",       "matrix":[ 4, 2], "x":2.5,   "y":4, "w":1.5 }, 
                {"label":"Space",     "matrix":[ 4, 6], "x":4,     "y":4, "w":7   }, 
                {"label":"Alt",       "matrix":[ 4,10], "x":11,    "y":4, "w":1.5 }, 
                {"label":"Left",      "matrix":[ 4,11], "x":13,    "y":4          }, 
                {"label":"Down",      "matrix":[ 4,12], "x":14,    "y":4          }, 
                {"label":"Right",     "matrix":[ 4,13], "x":15,    "y":4          },
```

File: keyboards/smithrune/iron165r2_h/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT , KC_PGDN,
+        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_UP  , KC_END ,
+        KC_LCTL, KC_LWIN, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_LEFT, KC_DOWN, KC_RGHT, KC_DEL 
+    ),
+    [1] = LAYOUT(
+        QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______
```
Comment (noroadsleft) [inline keyboards/smithrune/iron165r2_h/keymaps/default/keymap.c:33]:
```suggestion
        QK_GESC, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_BSPC, KC_DEL ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT , KC_PGDN,
        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_UP  , KC_END ,
        KC_LCTL, KC_LWIN, KC_LALT,                            KC_SPC ,                            KC_RALT,          KC_LEFT, KC_DOWN, KC_RGHT
    ),
    [1] = LAYOUT(
        QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                            _______,                            _______,          _______, _______, _______
```

File: keyboards/smithrune/iron165r2_h/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT , KC_PGDN,
+        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_UP  , KC_END ,
+        KC_LCTL, KC_LWIN, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_LEFT, KC_DOWN, KC_RGHT, KC_DEL 
+    ),
+    [1] = LAYOUT(
+        QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______
```
Comment (noroadsleft) [inline keyboards/smithrune/iron165r2_h/keymaps/via/keymap.c:33]:
```suggestion
        QK_GESC, KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_BSPC, KC_DEL ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT , KC_PGDN,
        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, KC_UP  , KC_END ,
        KC_LCTL, KC_LWIN, KC_LALT,                            KC_SPC ,                            KC_RALT,          KC_LEFT, KC_DOWN, KC_RGHT
    ),
    [1] = LAYOUT(
        QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                            _______,                            _______,          _______, _______, _______
```

File: keyboards/smithrune/iron165r2_s/config.h [hunk 5]
```diff
@@ -15,7 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#pragma once
+#define LED_CAPS_LOCK_PIN A4
```
Comment (noroadsleft) [inline keyboards/smithrune/iron165r2_s/config.h:18]:
```suggestion
#pragma once

#define LED_CAPS_LOCK_PIN A4
```

File: keyboards/smithrune/iron165r2_s/info.json [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+                {"label":"Ctrl",      "matrix":[ 4, 0], "x":0,     "y":4, "w":1.25}, 
+                {"label":"Win",       "matrix":[ 4, 1], "x":1.25,  "y":4, "w":1.25}, 
+                {"label":"Alt",       "matrix":[ 4, 2], "x":2.5,   "y":4, "w":1.25}, 
+                {"label":"Space",     "matrix":[ 4, 6], "x":3.75,  "y":4, "w":6.25}, 
+                {"label":"Alt",       "matrix":[ 4, 9], "x":10,    "y":4, "w":1.25}, 
+                {"label":"Win",       "matrix":[ 4,10], "x":11.25, "y":4, "w":1.25}, 
+                {"label":"Left",      "matrix":[ 4,11], "x":13,    "y":4          }, 
+                {"label":"Down",      "matrix":[ 4,12], "x":14,    "y":4          }, 
+                {"label":"Right",     "matrix":[ 4,13], "x":15,    "y":4          },
+                {"label":"Del",       "matrix":[ 4,14], "x":15,    "y":0          }
```
Comment (noroadsleft) [inline keyboards/smithrune/iron165r2_s/info.json:107]:
```suggestion
                {"label":"Del",       "matrix":[ 4,14], "x":15,    "y":0          } 
                {"label":"Tab",       "matrix":[ 1, 0], "x":0,     "y":1, "w":1.5 }, 
                {"label":"Q",         "matrix":[ 1, 1], "x":1.5,   "y":1          }, 
                {"label":"W",         "matrix":[ 1, 2], "x":2.5,   "y":1          }, 
                {"label":"E",         "matrix":[ 1, 3], "x":3.5,   "y":1          }, 
                {"label":"R",         "matrix":[ 1, 4], "x":4.5,   "y":1          }, 
                {"label":"T",         "matrix":[ 1, 5], "x":5.5,   "y":1          }, 
                {"label":"Y",         "matrix":[ 1, 6], "x":6.5,   "y":1          }, 
                {"label":"U",         "matrix":[ 1, 7], "x":7.5,   "y":1          }, 
                {"label":"I",         "matrix":[ 1, 8], "x":8.5,   "y":1          }, 
                {"label":"O",         "matrix":[ 1, 9], "x":9.5,   "y":1          }, 
                {"label":"P",         "matrix":[ 1,10], "x":10.5,  "y":1          }, 
                {"label":"{",         "matrix":[ 1,11], "x":11.5,  "y":1          }, 
                {"label":"}",         "matrix":[ 1,12], "x":12.5,  "y":1          }, 
                {"label":"|",         "matrix":[ 1,13], "x":13.5,  "y":1, "w":1.5 }, 
                {"label":"Page Up",   "matrix":[ 1,14], "x":15,    "y":1          }, 
                {"label":"Caps Lock", "matrix":[ 2, 0], "x":0,     "y":2, "w":1.75}, 
                {"label":"A",         "matrix":[ 2, 1], "x":1.75,  "y":2          }, 
                {"label":"S",         "matrix":[ 2, 2], "x":2.75,  "y":2          }, 
                {"label":"D",         "matrix":[ 2, 3], "x":3.75,  "y":2          }, 
                {"label":"F",         "matrix":[ 2, 4], "x":4.75,  "y":2          }, 
                {"label":"G",         "matrix":[ 2, 5], "x":5.75,  "y":2          }, 
                {"label":"H",         "matrix":[ 2, 6], "x":6.75,  "y":2          }, 
                {"label":"J",         "matrix":[ 2, 7], "x":7.75,  "y":2          }, 
                {"label":"K",         "matrix":[ 2, 8], "x":8.75,  "y":2          }, 
                {"label":"L",         "matrix":[ 2, 9], "x":9.75,  "y":2          }, 
                {"label":":",         "matrix":[ 2,10], "x":10.75, "y":2          }, 
                {"label":"\"",        "matrix":[ 2,11], "x":11.75, "y":2          }, 
                {"label":"~#",        "matrix":[ 2,12], "x":12.75, "y":2          }, 
                {"label":"Enter",     "matrix":[ 2,13], "x":13.75, "y":2, "w":1.25}, 
                {"label":"Page Down", "matrix":[ 2,14], "x":15,    "y":2          }, 
                {"label":"Shift",     "matrix":[ 3, 0], "x":0,     "y":3, "w":1.25},
                {"label":"\\|",       "matrix":[ 3, 1], "x":1.25,  "y":3          },
                {"label":"Z",         "matrix":[ 3, 2], "x":2.25,  "y":3          }, 
                {"label":"X",         "matrix":[ 3, 3], "x":3.25,  "y":3          }, 
                {"label":"C",         "matrix":[ 3, 4], "x":4.25,  "y":3          }, 
                {"label":"V",         "matrix":[ 3, 5], "x":5.25,  "y":3          }, 
                {"label":"B",         "matrix":[ 3, 6], "x":6.25,  "y":3          }, 
                {"label":"N",         "matrix":[ 3, 7], "x":7.25,  "y":3          }, 
                {"label":"M",         "matrix":[ 3, 8], "x":8.25,  "y":3          }, 
                {"label":"<",         "matrix":[ 3, 9], "x":9.25,  "y":3          }, 
                {"label":">",         "matrix":[ 3,10], "x":10.25, "y":3          }, 
                {"label":"?",         "matrix":[ 3,11], "x":11.25, "y":3          }, 
                {"label":"Shift",     "matrix":[ 3,12], "x":12.25, "y":3, "w":1.75}, 
                {"label":"Up",        "matrix":[ 3,13], "x":14,    "y":3          }, 
                {"label":"End",       "matrix":[ 3,14], "x":15,    "y":3          }, 
                {"label":"Ctrl",      "matrix":[ 4, 0], "x":0,     "y":4, "w":1.25}, 
                {"label":"Win",       "matrix":[ 4, 1], "x":1.25,  "y":4, "w":1.25}, 
                {"label":"Alt",       "matrix":[ 4, 2], "x":2.5,   "y":4, "w":1.25}, 
                {"label":"Space",     "matrix":[ 4, 6], "x":3.75,  "y":4, "w":6.25}, 
                {"label":"Alt",       "matrix":[ 4, 9], "x":10,    "y":4, "w":1.25}, 
                {"label":"Win",       "matrix":[ 4,10], "x":11.25, "y":4, "w":1.25}, 
                {"label":"Left",      "matrix":[ 4,11], "x":13,    "y":4          }, 
                {"label":"Down",      "matrix":[ 4,12], "x":14,    "y":4          }, 
                {"label":"Right",     "matrix":[ 4,13], "x":15,    "y":4          },
```

File: keyboards/smithrune/iron165r2_s/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    KC_CAPS,  KC_A,    KC_S,    KC_D,    KC_F,   KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS,  KC_ENTER,KC_PGDN,
+    KC_LSFT,  KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  KC_UP,   KC_END,
+    KC_LCTL , KC_LGUI, KC_LALT,                           KC_SPC,                    KC_RALT, KC_RGUI, KC_LEFT, KC_DOWN,  KC_RGHT, KC_DEL
+),
+[1] = LAYOUT(
+    QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______,                            _______,                   _______, _______, _______, _______, _______, _______
```
Comment (noroadsleft) [inline keyboards/smithrune/iron165r2_s/keymaps/default/keymap.c:31]:
```suggestion
    QK_GESC,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSPC, KC_BSPC, KC_DEL,
    KC_TAB,   KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,  KC_BSLS, KC_PGUP,
    KC_CAPS,  KC_A,    KC_S,    KC_D,    KC_F,   KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS,  KC_ENTER,KC_PGDN,
    KC_LSFT,  KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  KC_UP,   KC_END,
    KC_LCTL , KC_LGUI, KC_LALT,                           KC_SPC,                    KC_RALT, KC_RGUI,          KC_LEFT, KC_DOWN,  KC_RGHT
),
[1] = LAYOUT(
    QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______,                            _______,                   _______, _______,          _______, _______, _______
```

File: keyboards/smithrune/iron165r2_s/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    KC_CAPS,  KC_A,    KC_S,    KC_D,    KC_F,   KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS,  KC_ENTER,KC_PGDN,
+    KC_LSFT,  KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  KC_UP,   KC_END,
+    KC_LCTL , KC_LGUI, KC_LALT,                           KC_SPC,                    KC_RALT, KC_RGUI, KC_LEFT, KC_DOWN,  KC_RGHT, KC_DEL
+),
+[1] = LAYOUT(
+    QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______,                            _______,                   _______, _______, _______, _______, _______, _______
```
Comment (noroadsleft) [inline keyboards/smithrune/iron165r2_s/keymaps/via/keymap.c:31]:
```suggestion
    QK_GESC,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSPC, KC_BSPC, KC_DEL,
    KC_TAB,   KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,  KC_BSLS, KC_PGUP,
    KC_CAPS,  KC_A,    KC_S,    KC_D,    KC_F,   KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS,  KC_ENTER,KC_PGDN,
    KC_LSFT,  KC_NUBS, KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,  KC_UP,   KC_END,
    KC_LCTL , KC_LGUI, KC_LALT,                           KC_SPC,                    KC_RALT, KC_RGUI,          KC_LEFT, KC_DOWN,  KC_RGHT
),
[1] = LAYOUT(
    QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______,                            _______,                   _______, _______,          _______, _______, _______
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by Gondolindrim** [comment]
The merge conflict fix is too large and at this point there is no benefit in continuing this PR I feel. I will open a new one against current develop.

**Pull request closed without merging**